force_bdss/events/tests/test_data_source_events.py

```diff
@@ -25,9 +25,19 @@ def test_getstate_finish_event(self):
         self.assertDictEqual(
             event.__getstate__(),
             {
-                "model_data": {},
+                "model_data": {"output_names": []},
                 "id": "force_bdss.events.data_source_events."
                 "DataSourceFinishEvent",
             },
         )
         self.assertIsInstance(event, MCORuntimeEvent)
+
+    def test_serialize_finish_event(self):
+        event = DataSourceFinishEvent()
+
+        self.assertListEqual(event.serialize(), [])
+
+    def test_serialize_start_event(self):
+        event = DataSourceStartEvent()
+
+        self.assertListEqual(event.serialize(), [])
```

Comment:
Can we also provide tests to check the expected outcome should be when `event.input_names` or `event.output_names` are assigned values?